db/migrate/20231215185924_create_stays.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStays < ActiveRecord::Migration[7.0]
+  def change
+    create_table :stays do |t|
+      t.references :room, null: false, foreign_key: true
+      t.references :guest, null: false, foreign_key: true
```

Comment:
This setup look great for the joins table. I don't see it reflected in the schema though so make sure you run `rails db:migrate` to see the schema.rb change